src/init.js

```diff
@@ -84,3 +85,44 @@ EventBus.$on('signaling-recording-status-changed', (token, status) => {
 		showError(t('spreed', 'The recording failed. Please contact your administrator.'))
 	}
 })
+
+/**
+ * Migrate localStorage to @nextcloud/browser-storage
+ *
+ * In order to preserve the user settings while migrating to the abstraction,
+ * we loop over the localStorage entries and add the matching ones to the
+ * BrowserStorage
+ */
+const migrateDirectLocalStorageToNextcloudBrowserStorage = () => {
+	const storageKeys = Array.from(Array(localStorage.length), (i) => localStorage.key(i)).filter(key => key.startsWith('audioDisabled_')
+		|| key.startsWith('videoDisabled_')
+		|| key.startsWith('virtualBackgroundEnabled_')
+		|| key.startsWith('virtualBackgroundType_')
+		|| key.startsWith('virtualBackgroundBlurStrength_')
+		|| key.startsWith('virtualBackgroundUrl_')
+	)
```

Comment:
A very minor performance note: Web Storage API is a synchronous Browser API. Reading 300-400 entries takes about 0.2ms of blocking time. Even without `filter` (no matter if `Object.keys` or `localStorage.key` is used).

An alternative could be to store only one flag with `isMigrated` or migration version.